cmd/aries-agent-rest/go.mod

```diff
@@ -21,3 +21,8 @@ require (
 )
 
 go 1.15
+
+replace (
+	github.com/kilic/bls12-381 => github.com/trustbloc/bls12-381 v0.0.0-20201008080608-ba2e87ef05ef
+	github.com/phoreproject/bls => github.com/trustbloc/bls v0.0.0-20201008085849-81064514c3cc
+)
```

Comment:
We can combine these replaces with the one in line 7.

Reply:
fixed